stac_mjx/stac.py

```diff
@@ -390,7 +390,7 @@ def _package_data(self, mjx_model, q, x, walker_body_sites, kp_data, batched=Fal
             x = x.reshape(-1, x.shape[-1])
             q = q.reshape(-1, q.shape[-1])
         else:
-            offsets = self._offsets
+            offsets = self._offsets.reshape((-1, 3))
```

Comment:
That's what I figured, but I wanted to make, since a few things were changed by the PR that introduced it. 

Just to double check, the other branch of this doesn't need it as well?